crates/env/src/engine/off_chain/impls.rs

```diff
@@ -333,6 +339,21 @@ impl EnvBackend for EnvInstance {
         Ok(())
     }
 
+    fn sr25519_verify(
+        &mut self,
+        signature: &[u8; 64],
+        message: &[u8],
+        pub_key: &[u8; 32],
+    ) -> Result<()> {
+        let context = signing_context(b"substrate");
```

Comment:
For further use, i think there should be an option to verify signature from any context. Right now there is such limitation.

Reply:
Could you create a `SIGNING_CTX` constant like in the substrate and add a comment with the link to the code on the substrate side(that substrate uses the same context)?